CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
+# 3.0.0-rc.6 (2019/12/04)
+
+- Fixed the keyboard navigation so that hitting enter while editing a cell saves it and starts editing the next in the column. [#20](https://github.com/blackbaud/skyux-ag-grid/pull/20)
```

Comment:
I'll let docs chime in here, but how about something like,
`Fixed the ag-Grid service so the enter key will save a cell and place focus in the next cell.`

Also, you'll need to update the date above ☝️ .